src/cli.ts

```diff
@@ -71,7 +72,7 @@ function usage(msg?: string): void {
 async function run(verb: string, files: string[]): Promise<boolean> {
   const options: Options = {
     dryRun: cli.flags.dryRun || false,
-    gtsRootDir: `${process.cwd()}/node_modules/gts`,
+    gtsRootDir: path.resolve(__dirname, '../..'),
```

Comment:
I was having a hard time understanding this. Worth commenting that `cli.js` is installed under `{root}/build/src/`?

Reply:
This is valid criticism. However, I think this happens pervasively across the codebase, include when the `package.json`, `tslint.json` and other files are loaded. Adding a comment here but not elsewhere doesn't seem right, and neither does it make sense to have these littered throughout the code-base.

Perhaps we should switch the directory structure that doesn't have this problem (i.e. having a toplevel `ts/` directory that would contain `src` and `test`). WDYT?

Reply:
Yeah, understood. Having `src` and `test` under `ts` would help but I don't think it's a high priority.